### ISSUE TITLE
Odd Even

### ISSUE THREAD

**Issue by abirbhattacharya82** [abirbhattacharya82/Hacktoberfest22-DSA-CP-Code-Dump#21]
Write a Program in Java to check if a Number is Odd or Even

**Comment by Dwaipayan-Maz** [comment]
Kindly assign me this issue.

**Comment by harmanjit356** [comment]
Kindly assign me this issue.

**Comment by poseidon2060** [comment]
hey is this issue open? Can you please assign it to me if it is?

**Comment by abirbhattacharya82** [comment]
Hi Guys @harmanjit356, @poseidon2060 this issue is already assigned. If you want to implement the same in other languages except JAVA feel free to open an Issue

**Comment by harmanjit356** [comment]
Okay 👍

**Comment by sabitak27** [comment]
Hi, I can implement this code in C++.

**Comment by abirbhattacharya82** [comment]
Yeah sure. Make a separate issue regarding it. I will assign it to you
